enterprise-metrics/docs/README.md

```diff
@@ -297,6 +327,62 @@ $ kubectl create secret generic gem-license -from-file=license.jwt
 
 `service` is the Kubernetes Service for the admin-api.
 
+## obj alertmanager
+
+`alertmanager` has configuration for the alertmanager. To disable the alertmanager, ensure the ruler object field is hidden
```

Comment:
Should it say 'ruler object'?

Reply:
Good catch, these docs are generated from the `main.libsonnet` file using `make docs/README.md` so I will fix it there and regenerate